app/models/conference_preference.rb

```diff
@@ -1,5 +1,21 @@
 class ConferencePreference < ActiveRecord::Base
+  validates :terms_accepted?, acceptance: true
```

Comment:
@juuh42dias I'm not sure I understood Max's solution @klappradla  "only on create" means on create of a team? It won't be very convenient in the cases when we want to edit something else but the conference preferences. E.g. AFAIU we always show the conference block on the team view/edit pages, regardless of the current phase (applications, coding summer etc.). It means that we will have to accept the conference terms when we edit a team during the application phase, right? Even if we fix this and enable the conference functionality only for the accepted teams (which we should probably do in a separate issue), there will be time when we don't have a list of conferences yet, but already demand to accept the terms the moment a team wants to add some team info (e.g. blog url). 
I love the idea that we check the terms only if a conference is selected. On the other hand, I understand why Max @klappradla proposes what he proposes. After all, if we make the terms validation independent of the conference preferences, it won't be the end of the world, if we hide this for applying teams (at least). What do you think @klappradla ?